client/js/widgets/pile.js

```diff
@@ -75,10 +75,29 @@ class Pile extends Widget {
 
   async click(mode='respect') {
     if(!await super.click(mode)) {
-      $('#pileOverlay').innerHTML = `<p>${this.handle.textContent} cards</p><p>Drag the handle with the number to drag the entire pile.</p>`;
+
+
+//       const modalDiv = document.createElement('div');
+//       modalDiv.classList.add = 'modal';
+//       $('#pileOverlay')
+//
+//       const modalTitle = document.createElement('div');
+//       modalTitle.classList.add = 'inputtitle';
+//       var thisField = { type: 'title',text: ${this.handle.textContent} cards+' cards' };
+//       formField(thisField, modalTitle, 'pileTitle')
+//       modalDiv.appendChild(modalTitle);
+
+
+
```

Comment:
What's the plan here? You marked it as "ready to review" but this doesn't look finished yet. Not that that's a problem; I just don't know how to review this. :smile: